controllers/reviews.controllers.js

```diff
@@ -7,3 +7,12 @@ exports.getReviews = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getReviewById = (req, res, next) => {
+  const { review_id } = req.params;
+  selectReviewById(review_id)
+    .then((selectedReview) => {
+      res.status(200).send({ selectedReview });
```

Comment:
I'd recommend just returning on a key of 'review' for this one, but selectedReview is okay.